### ISSUE TITLE
Cannot compiled due to transitive dependency on zeroize

### ISSUE THREAD

**Issue by gim913** [guoxbin/dtool#25]
Last version of dtool depends on [schnorrkel 0.8.5](https://crates.io/crates/schnorrkel/0.8.5/dependencies) which depends on zeroize 0.9.x, it seems all zeroize packages < 1.0 have been purged.

```
error: failed to compile `dtool v0.10.1`, intermediate artifacts can be found at `C:\Users\gim91\AppData\Local\Temp\cargo-installjjerSE`

Caused by:
  failed to select a version for the requirement `zeroize = "^0.9.3"`
  candidate versions found which didn't match: 1.3.0, 1.2.0, 1.1.1, ...
  location searched: crates.io index
  required by package `schnorrkel v0.8.5`
      ... which is depended on by `dtool v0.10.1`
```

**Comment by guoxbin** [comment]
@gim913  What's your rustc version

**Comment by gim913** [comment]
```
installed toolchains
--------------------

stable-x86_64-pc-windows-msvc (default)
nightly-x86_64-pc-windows-msvc

active toolchain
----------------

stable-x86_64-pc-windows-msvc (default)
rustc 1.48.0 (7eac88abb 2020-11-16)
```

**Comment by schoetbi** [comment]
Just checked with `rustc 1.52.1 (9bc8c42bb 2021-05-09) (from rustc 1.51.0 (2fd73fabe 2021-03-23))` on Windows 10 same result:

```
C:\>cargo install dtool
    Updating crates.io index
  Installing dtool v0.10.1
error: failed to compile `dtool v0.10.1`, intermediate artifacts can be found at `C:\Users\schoetbi\AppData\Local\Temp\cargo-installIC8qd9`

Caused by:
  failed to select a version for the requirement `zeroize = "^0.9.3"`
  candidate versions found which didn't match: 1.3.0, 1.2.0, 1.1.1, ...
  location searched: crates.io index
  required by package `schnorrkel v0.8.5`
      ... which is depended on by `dtool v0.10.1`
```

**Comment by guoxbin** [comment]
Could you help check if the latest version fixed the compiling issue? @schoetbi @gim913 
Sorry for been late.

**Comment by schoetbi** [comment]
@guoxbin : The error from this issue is fixed but now I get this (dtool v0.11.0):
```

   Compiling regex-syntax v0.6.25
error[E0658]: arbitrary expressions in key-value attributes are unstable
  --> C:\Users\schoetbi\.cargo\registry\src\github.com-1ecc6299db9ec823\subtle-ng-2.5.0\src\lib.rs:12:10
   |
12 | #![doc = include_str!("../README.md")]
   |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #78835 <https://github.com/rust-lang/rust/issues/78835> for more information

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
error: could not compile `subtle-ng`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: failed to compile `dtool v0.11.0`, intermediate artifacts can be found at `C:\Users\schoetbi\AppData\Local\Temp\cargo-installLXN2iB`

Caused by:
  build failed
```

**Comment by enizor** [comment]
For anyone getting the same error: update your rust compiler, it was added by [Rust 1.54](https://blog.rust-lang.org/2021/07/29/Rust-1.54.0.html#attributes-can-invoke-function-like-macros)
